beets/ui/commands.py

```diff
@@ -810,6 +810,10 @@ def resolve_duplicate(self, task, found_duplicates):
         elif sel == 'k':
             # Keep both. Do nothing; leave the choice intact.
             pass
+        elif sel == 'm':
+            # Merge the two together
```

Comment:
Style nitpick: for comments like this, please use a period at the end of sentences.